### PR TITLE
Add install page

### PR DESCRIPTION
**Pull request by WetDesertRock**
Completes #133

### CODE REVIEW

**Comment by WetDesertRock** [comment]
Fixes #133

File: php/install_page.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	$configData = "<?php
+\$dbAddress = \"$safeHost\";
+\$dbUsername = \"$safeUser\";
+\$dbPassword = \"$safePassword\";
+\$dbDatabaseName = \"$safeName\";
+?>";
+	
+	$result = file_put_contents($dbConfigPath, $configData);
+	if ($result == FALSE) {
+		return "Unable to write config file. Please manually fill it out (database has been initialized)";
```
Comment (liambaloh) [inline php/install_page.php]: Please add $dbConfigPath in this output so the user knows where to look for / create the config file to be filled

File: php/page.php [hunk 2]
```diff
@@ -1,5 +1,4 @@
 <?php
-$templateBasePath = "template/";
```
Comment (liambaloh) [inline php/page.php]: Intentional? The reason for this is to eventually allow different site templates - so it'd look like `template/template1/index.html`. At the moment, everything is directly in `template/` though

Comment (WetDesertRock) [reply php/page.php]: Ahha, that makes sense. I moved it to global.php but I think knowing this helps me understand that it's okay to just copy that value out and put it directly into the install code.

**Comment by WetDesertRock** [comment]
Made the changes as discussed. Note that once the database migration change is merged this will have to have minor fixes added.

**Pull request merged**